grafana/dashboards/cmode/health.json

```diff
@@ -3185,11 +3185,11 @@
         "options": [
           {
             "selected": true,
-            "text": "Provides EMS events selected Time Range configured in ems.yaml for EMS collector",
-            "value": "Provides EMS events selected Time Range configured in ems.yaml for EMS collector"
+            "text": "The EMS collector gathers EMS events as defined in your ems.yml file. This panel specifically displays events with emergency severity that occurred within a selected time range.",
+            "value": "The EMS collector gathers EMS events as defined in your ems.yml file. This panel specifically displays events with emergency severity that occurred within a selected time range."
           }
         ],
-        "query": "Provides EMS events selected Time Range configured in ems.yaml for EMS collector",
+        "query": "The EMS collector gathers EMS events as defined in your ems.yml file. This panel specifically displays events with emergency severity that occurred within a selected time range.",
```

Comment:
The EMS collector gathers EMS events as defined in your ems.yml file. This panel displays events with emergency severity that occurred within the selected time range.